### PR TITLE
Pm 1463 update origin notification vpos

### PR DESCRIPTION
**Pull request by albertogelmi-sia**
#### List of Changes

the origin field is made dynamic in the VPos' method phase

#### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have created or updated an API interface.
  - [ ] I have updated the OpenAPI spec
- [ ] I have created or updated a database table
  - [ ] I have updated both the Oracle and Postgres migrations
- [X] My PR is cohesive and self-contained (rule of thumb: less than 250 changed lines) 

### Link to story

https://pagopa.atlassian.net/browse/PM-1463

### CODE REVIEW

File: src/main/resources/templates/methodNotification.html [hunk 1]
```diff
@@ -1,10 +1,15 @@
...
-		var getOrigin = window.location.origin;
-		window.parent.postMessage("3DS.Notification.Received", getOrigin);
-	</script>
-</head>
-<body></body>
+<html xmlns:th="http://www.thymeleaf.org">
+<body>
+
+    <script th:inline="javascript">
+        /*<![CDATA[*/
```
Comment (giovanniberti) [inline src/main/resources/templates/methodNotification.html:6]: Why is CDATA here? In an HTML  document it should not be needed

File: src/main/resources/templates/methodNotification.html [hunk 2]
```diff
@@ -1,10 +1,15 @@
...
+
+        var getOrigin = /*[[${origin}]]*/;
```
Comment (giovanniberti) [inline src/main/resources/templates/methodNotification.html:8]: Shouldn't it be

```suggestion
        var getOrigin = [[${origin}]];
```
?

File: src/main/resources/templates/methodNotification.html (hunk 1 above)
Comment (albertogelmi-sia) [reply src/main/resources/templates/methodNotification.html]: CDATA removed

File: src/main/resources/templates/methodNotification.html (hunk 2 above)
Comment (albertogelmi-sia) [reply src/main/resources/templates/methodNotification.html]: CDATA removed

**Review by giovanniberti — APPROVED** [review]

**Pull request merged**